egs/wsj/s5/steps/nnet3/tdnn/train_raw.sh

```diff
@@ -0,0 +1,636 @@
+#!/bin/bash
```

Comment:
I think the script should be named as <code>train_rawnnet_densetargets.sh</code>. Since there is another effort on raw features things can get pretty confusing in the future.